DayrSuryan/DSEthiop7.xml

```diff
@@ -343,6 +343,23 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            </q>
                         </item>
 
+                        <item xml:id="a3">
```

Comment:
If you decided now for an addition, it would be good to delete the colophon, right now I think we have both